### DISCUSSION TITLE
Switch from Pi-Hole to AdGuard

### DISCUSSION THREAD

**Discussion by HyperCriSiS** [AdguardTeam/AdGuardHome#5983] [Q&A]
Hi,

I just started to switch to AdGuard home and I have a hard time importing the Pi-HoledData.
I wonder there seems to be no convenient way to bulk add lists and so on?
How can I automate this process more or do bulk imports?
Adding everything one by one is a hardcore annoying task which will take many hours.

Thanks and regards
Hyper

**Comment by Kapujino** [comment]
I'm also migrating from pihole to adguard and it looks like there is no easy way to migrate all the blocklists.
However, I managed to migrate all my blocklists to adguard and documented it here:
https://gist.github.com/Kapujino/03feaa159f72c84bbef37f3551cb069a
